doc/versions.json

```diff
@@ -13,9 +14,15 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.3.99",
-      "nrfxlib": "1.3.99",
-      "zephyr": "2.3.99",
+      "ncs": "1.4.99",
+      "nrfxlib": "1.4.99",
+      "zephyr": "2.4.99",
```

Comment:
@ru-fu , should we use 2.4.99 here? The upmerge is planned after 1.4.0 release...

Reply:
No, that should be updated with the upmerge. 
@carlescufi Is there some checklist to make sure to update this file as well?

Reply:
Correct, this should be left at 2.4.0, which is the current zephyr version. On the next upmerge this needs to be updated. I will add a reminder for this in the upmerge page.